pex/bin/pex.py

```diff
@@ -503,8 +514,12 @@ def main():
     args, cmdline = args, []
 
   options, reqs = parser.parse_args(args=args)
+  if options.pex_root:
```

Comment:
should we be doing the inverse here (e.g. set options.pex_root from ENV default) in the case that `options.pex_root` is `None`?